tests/unit-tests/gpartition/test_hard_negative_post_partition.py

```diff
@@ -0,0 +1,114 @@
+"""
+    Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+import os
+import json
+import torch as th
+import numpy as np
+from typing import Dict
+
+import pytest
+
+from numpy.testing import assert_almost_equal
+from graphstorm.model.utils import load_dist_nid_map
+from dgl.data.utils import load_tensors, save_tensors
+from graphstorm.gpartition.post_hard_negative import (shuffle_hard_negative_nids,
+                                                      load_hard_negative_config)
+
+
+def test_load_hard_negative_config():
+    # For config with hard negative transformation
+    json_file_path = (f"/graphstorm/tests/unit-tests/gpartition/"
+                      f"config/gsprocessing_hard_negative_config.json")
```

Comment:
Use paths relative to the location of the test scripts, usually you'd add the static files in a `resources` dir like we do here:

https://github.com/awslabs/graphstorm/blob/19fac3ba136e77c404c551cb59d53b6f79a5b2c3/graphstorm-processing/tests/conftest.py#L34

https://github.com/awslabs/graphstorm/blob/19fac3ba136e77c404c551cb59d53b6f79a5b2c3/graphstorm-processing/tests/conftest.py#L80-L85